libs/utils/energy_model.py

```diff
@@ -77,6 +80,18 @@ class ActiveState(namedtuple('ActiveState', ['capacity', 'power'])):
     def __new__(cls, capacity=None, power=None):
         return super(ActiveState, cls).__new__(cls, capacity, power)
 
+    # helpers for yaml serialization
+    yaml_tag = u'em_active_state:capacity,power'
+
+    @classmethod
+    def to_yaml(cls, representer, node):
+        return representer.represent_scalar(cls.yaml_tag,
+                u'{.capacity},{.power}'.format(node, node))
+
+    @classmethod
+    def from_yaml(cls, constructor, node):
+        return cls(*node.value.split(','))
```

Comment:
This may work, but there is a good chance that the rest of the code is expecting integers instead of strings, so something like that:
cap, power = (int(x) for x in node.value.split(','))
return cls(capacity=cap, power=power)